.circleci/config.yml

```diff
@@ -28,7 +28,6 @@ templates:
           # use master's cache but don't pollute it back.
           - v2-godeps-{{ .Branch }}-{{ .Revision }}
           - v2-godeps-{{ .Branch }}-
-          - v2-godeps-master-
```

Comment:
hi @truthbk ,

Why remove this line? It allows to pre-seed a branch's cache by `master`. Did you run into cache issues?